bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -530,8 +531,10 @@ CGINS-CAMDSA-00122,,Sensor,0,Underwater Digital Still Camera: CAMDS Series A,CAM
 CGINS-CAMDSB-00100,N00694,Sensor,0,Underwater Digital Still Camera: CAMDS Series B,CAMDS,Kongsberg Underwater Technology Inc,Custom,0484-6002-0100,,,,,
 CGINS-CTDAVN-00103,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,103,,,,CGSN AUV Instrument Pool,
 CGINS-CTDAVN-00104,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,104,,,,CGSN AUV Instrument Pool,
-CGINS-CTDAVN-00424,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,424,,,,CGSN AUV Instrument Pool,
-CGINS-CTDAVN-00425,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Sea-Bird Electronics,SBE AUV Payload CTD,425,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00424,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,424,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00425,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,425,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00526,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,526,,,,CGSN AUV Instrument Pool,
+CGINS-CTDAVN-00527,,Sensor,1,CTD for AUV: CTD Series N,CTDGV for Hydroid AUV,Neil Brown Ocean Sensors Inc,NBOSI AUV Payload CTD,527,,,,CGSN AUV Instrument Pool,

```

Comment:
updates to 424 and 425 look good, and 526 and 527 new additions look good

Reply:
Looks good.